bazel/envoy_build_system.bzl

```diff
@@ -210,6 +211,7 @@ envoy_select_wasm_cpp_tests = _envoy_select_wasm_cpp_tests
 envoy_select_wasm_rust_tests = _envoy_select_wasm_rust_tests
 envoy_select_wasm_wavm = _envoy_select_wasm_wavm
 envoy_select_wasm_wasmtime = _envoy_select_wasm_wasmtime
+envoy_select_wasm_wamr = _envoy_select_wasm_wamr
```

Comment:
Nit: move up to sort alphabetically.

Reply:
OK, Should I move `envoy_select_wasm_v8 = _envoy_select_wasm_v8` up to sort alphabetically together? LOL

Reply:
If you could, thanks and sorry for nitpicking!

I'm actually surprised that format check doesn't complain about this, I believe it used to in the past.

Reply:
LOL. Fixed.